app/Http/Controllers/ShopOwnerController.php

```diff
@@ -46,12 +47,12 @@ public function list(){
     public function show($id){
         $orderItems = OrderItem::where('order_id', Auth::user()->owner_shop_id);
 
-        // dd($orderItems->get());
-
         $order = Order::find($id);
+        $address = Address::where('order_id', $id)->get();
         // dd($order);
```

Comment:
This is not correct. As order can have 1 or more addresses. One for billing, one for shipping. You should get the address by using eloquent relationship. Check eloquent "with" function